utils/ssh/utils.go

```diff
@@ -60,18 +60,21 @@ func fileExist(path string) bool {
 }
 
 func ReadPipe(stdout, stderr io.Reader, alsoToStdout bool) {
-	var combineSlice []string
-	var combineLock sync.Mutex
+	var combineDoneSlice []string
+	var combineDoneLock sync.Mutex
+	var combineErrSlice []string
```

Comment:
good , if we want to see specific error, `combineErrSlice` will contain all messages.

Reply:
Maybe we can pass a flag and filter to readPipe(), then we can select error
such as:
```
if strings.Contains(message, errorFilter) {
			fmt.Println("Specific error:", message)
		}
```

Reply:
yes, agree with you.@Daz-3ux